featuretools/feature_base/feature_base.py

```diff
@@ -328,21 +328,23 @@ def unique_name(self):
 class IdentityFeature(FeatureBase):
     """Feature for entity that is equivalent to underlying variable"""
 
-    def __init__(self, variable):
+    def __init__(self, variable, name=None):
         entity_id = variable.entity_id
         self.variable = variable.entityset.metadata[entity_id][variable.id]
         self.return_type = type(variable)
         super(IdentityFeature, self).__init__(entity=variable.entity,
                                               base_features=[],
                                               relationship_path=[],
                                               primitive=PrimitiveBase)
+        if name is not None:
```

Comment:
Since each feature subclass has this same logic in their respective init we could pass `name` to the FeatureBase `__init__` call and handle this there